features/step_definitions/my_steps.rb

```diff
@@ -13,12 +13,25 @@
 
 Given(/^I upload a spreadsheet like this:$/) do |table|
   mediators_data_table = DataHelpers::MediatorsDataTable.new(table.raw)
-  with_practice_data =DataHelpers:: PracticeData.new(mediators_data_table)
+  with_practice_data = DataHelpers::PracticeData.new(mediators_data_table)
   upload_spreadsheet(with_practice_data.headings, with_practice_data.data)
 end
 
 Given(/^I upload a mediator with practice data (.*)/) do |practice_data|
   mediators_data_table = DataHelpers::MediatorsDataTable.create_mediator()
   with_practice_data = DataHelpers::PracticeData.new(mediators_data_table, practice_data: practice_data)
   upload_spreadsheet(with_practice_data.headings, with_practice_data.data)
-end
\ No newline at end of file
+end
+
+Given(/^I have a spreadsheet like this:$/) do |table|
```

Comment:
This is nearly the same as this step:

```ruby
Given(/^I upload a spreadsheet like this:$/) do |table|
  mediators_data_table = DataHelpers::MediatorsDataTable.new(table.raw)
  with_practice_data = DataHelpers::PracticeData.new(mediators_data_table)
  upload_spreadsheet(with_practice_data.headings, with_practice_data.data)
end
```

Looks like an opportunity to DRY it up.